packages/ibm-products/src/components/FullPageError/FullPageError.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Grid, Column } from '@carbon/react';
+import { ErrorGenericSVG } from './assets/ErrorGenericSVG';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg /*, carbon */ } from '../../settings';
+
+// Carbon and package components we use.
+/* TODO: @import(s) of carbon components and other package components. */
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--full-page-error`;
+const componentName = 'FullPageError';
+
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+// Default values can be included here and then assigned to the prop params,
+// e.g. prop = defaults.prop,
+// This gathers default values together neatly and ensures non-primitive
+// values are initialized early to avoid react making unnecessary re-renders.
+// Note that default values are not required for props that are 'required',
+// nor for props where the component can apply undefined values reasonably.
+// Default values should be provided when the component needs to make a choice
+// or assumption when a prop is not supplied.
+
+// Default values for props
+// const defaults = {
+//   /* TODO: add defaults for relevant props if needed */
+// };
+
+/**
+ * TODO: A description of the component.
+ */
+export let FullPageError = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+
+      children /* TODO: remove if not needed. */,
+      className,
+      description,
+      errorLabel,
+      title,
+      /* TODO: add other props for FullPageError, with default values if needed */
+
+      // Collect any other property values passed in.
+      ...rest
+    },
+    ref
+  ) => {
+    return (
+      <div
+        {
+          // Pass through any other property values as HTML attributes.
+          ...rest
+        }
+        className={cx(
+          blockClass, // Apply the block class to the main HTML element
+          className, // Apply any supplied class names to the main HTML element.
+          // example: `${blockClass}__template-string-class-${kind}-n-${size}`,
+          {
+            // switched classes dependant on props or state
+            // example: [`${blockClass}__here-if-small`]: size === 'sm',
+          }
+        )}
+        ref={ref}
+        role="main"
+        {...getDevtoolsProps(componentName)}
+      >
+        <div className={`${blockClass}__error-container`}>
+          <Grid className={`${blockClass}__error-grid`}>
+            <Column
+              sm={4}
+              md={3}
+              lg={6}
+              className={`${blockClass}__error-column`}
+            >
+              {errorLabel && (
+                <p className={`${blockClass}__error-label`}>{errorLabel}</p>
+              )}
```

Comment:
Since the `errorLabel` is required already, we can probably simplify to just ...

```suggestion
                <p className={`${blockClass}__error-label`}>{errorLabel}</p>
```